### PR TITLE
DEV-3188-feat(halfslice): add test/solution for Go exam halfslice

### PR DESCRIPTION
**Pull request by tiagocollot**
Added test and solution for Go exam halfslice

### CODE REVIEW

File: tests/halfslice_test/main.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests-training/solutions"
```
Comment (jotapero) [inline tests/halfslice_test/main.go:7]: Fix: "github.com/01-edu/go-tests/solutions"

**Review by jotapero — CHANGES_REQUESTED** [review]

File: tests/halfslice_test/main.go (hunk 1 above)
Comment (tiagocollot) [reply tests/halfslice_test/main.go]: Thank you José for noticing it. It is fixed!

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**